Public/Src/Engine/Scheduler/PipExecutor.cs

```diff
@@ -934,6 +934,40 @@ private static void MakeSharedOpaqueOutputIfNeeded(IPipExecutionEnvironment envi
             }
         }
 
+        /// <summary>
+        /// Report change affected ouputs to file content manager
+        /// </summary>
+        internal static void ReportSourceChangeAffectedOutputs(

```

Comment:
>ReportSourceChangeAffectedOutputs [](start = 29, length = 33)

Who's calling this method?

Reply:
In the scheduler, postprocess and copyfile.